src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheck.java

```diff
@@ -121,49 +192,55 @@
     /** Lines logged as having incorrect indentation. */
     private Set<Integer> incorrectIndentationLines;
 
-    /** How many tabs or spaces to use. */
+    /** Specify how far new indentation level should be indented when on the next line. */
     private int basicOffset = DEFAULT_INDENTATION;
 
-    /** How much to indent a case label. */
+    /** Specify how far a case label should be indented when on next line. */
     private int caseIndent = DEFAULT_INDENTATION;
 
-    /** How far brace should be indented when on next line. */
+    /** Specify how far a braces should be indented when on the next line. */
     private int braceAdjustment;
 
-    /** How far throws should be indented when on next line. */
+    /** Specify how far a throws clause should be indented when on next line. */
     private int throwsIndent = DEFAULT_INDENTATION;
 
-    /** How much to indent an array initialization when on next line. */
+    /** Specify how far an array initialisation should be indented when on next line. */
     private int arrayInitIndent = DEFAULT_INDENTATION;
 
-    /** How far continuation line should be indented when line-wrapping is present. */
+    /** Specify how far continuation line should be indented when line-wrapping is present. */
     private int lineWrappingIndentation = DEFAULT_INDENTATION;
 
     /**
-     * Force strict condition in line wrapping case. If value is true, line wrap indent
-     * have to be same as lineWrappingIndentation parameter, if value is false, line wrap indent
-     * have to be not less than lineWrappingIndentation parameter.
+     * Force strict indent level in line wrapping case. If value is true, line wrap indent
+     * have to be same as lineWrappingIndentation parameter. If value is false, line wrap indent
+     * could be bigger on any value user would like.
      */
     private boolean forceStrictCondition;
 
     /**
-     * Get forcing strict condition.
+     * Getter to query strict indent level in line wrapping case. If value is true, line wrap indent
```

Comment:
Should we be validating getters for a check?
The current code only validates setters, not getters.
https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java#L538-L539

Reply:
Getters in Checks should not exist at all, if present, it should not be part of xdoc, we can skip them

Reply:
@pbludov It seems we shouldn't change the getters.

Reply:
we can keep changing getters, it does not matter, what ever is easier.